src/components/header/_macro-options.md

```diff
@@ -25,6 +26,23 @@
 | large   | HTML   | true     | Any HTML to render an image for example embedded `<svg>` or `<img>`            |
 | small   | HTML   | false    | Optionally provide a version of the logo more suited to mobile viewports       |
 
+## MastheadLogoMulti
+
+| Name    | Type                         | Required | Description                                                                    |
+| ------- | ---------------------------- | -------- | ------------------------------------------------------------------------------ |
+| classes | string                       | false    | Classes to be added. Helpful to add a margin utility class to control spacing. |
+| logo1   | object`<MastheadLogoObject>` | true     | First Logo                                                                     |
+| logo2   | object`<MastheadLogoObject>` | false    | Second Logo                                                                    |
+| logo3   | object`<MastheadLogoObject>` | false    | Third Logo                                                                     |
+
+## MastheadLogoObject
+
+| Name           | Type           | Required | Description                                                                                                                                                                                                                         |
+| -------------- | -------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| logoImage      | HTML or String | false    | Any HTML to render an image for example embedded `<svg>` or `<img>`. Use 'ONS Logo' for the default ONS icon. If logoImageLarge is provided, this will be used for smaller screens.                                                 |
```

Comment:
```suggestion
| logoImage      | HTML or string | false    | Any HTML to render an image for example embedded `<svg>` or `<img>`. Use 'ONS Logo' for the default ONS icon. If logoImageLarge is provided, this will be used for smaller screens.                                                 |
```